Classes/LinkHandler/GitHubLinkHandler.php

```diff
@@ -55,7 +55,6 @@ public function __construct(
     /**
      * Initialize the handler
      *
-     * @param AbstractLinkBrowserController $linkBrowser
      * @param string $identifier
```

Comment:
When I asked in #typo3-cms-coredev Stefan Bürk told me either all Parameters should be mentioned in the comment or none at all: https://typo3.slack.com/archives/C03AM9R17/p1665995703004209

Reply:
I disagree: If a docblock for a param holds valuable information, then it should be there. If not, it should be removed as it is redundant. Here, $linkBrowser is type-hinted, whereas $identifier is not. We should type-hint it also (then the docblock is redundant is should be removed), but this is not in the scope of this PR.

Reply:
Uppon implementing an interface we have to stick to the same type-hints as provided by the interface. Therefore we cannot type-hint $identifier for now. Also I would rather stick to the Core best-practices here then developing new one. See https://github.com/TYPO3/typo3/blob/a9315819068262c906c8987567a86e6a29c70016/typo3/sysext/backend/Classes/LinkHandler/LinkHandlerInterface.php#L55

Reply:
Then I have to define a skip section in rector.php for that class. So the param docblocks rule from rector is not applied anymore to this class. Alternatively, I can use a `@noRector` docblock above the `@param` which looks more weird in code snippets for the docs.

Reply:
Ok, so there is no rector rule saying no or all params should be commented? Maybe we should carry the discussion to #ext-typo3-rector? This might be a missing feature. Or the Core way might be wrong...

Reply:
For now I am ok with removing the not-needed params